src/index.js

```diff
@@ -9,8 +9,8 @@ import Runner from 'jscodeshift/dist/Runner.js';
 import util from './util';
 import chalk from 'chalk';
 
-function runTransforms({silent}, transforms, files) {
-	return Promise.mapSeries(transforms, transform => Runner.run(transform, files, {silent: silent}));
+function runTransforms(settings, transforms, files) {
```

Comment:
@greg-a-smith thanks for the PR, can you have it default it to silent but allow override?

Reply:
Just to clarify, are you asking this as a new request?  Because before it was simply pruning the `silent` value from the parameter object and then passing it as the only property in an object to `Runner.run`.  So this change shouldn't have altered the behavior of `silent` (unless I'm missing something).

Reply:
my bad, i misread.  all good. merging